AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/ViewComponents/VideoView/LocalVideoViewModel.swift

```diff
@@ -51,11 +51,20 @@ class LocalVideoViewModel: ObservableObject {
         if localVideoStreamId != localUserState.localVideoStreamIdentifier {
             localVideoStreamId = localUserState.localVideoStreamIdentifier
         }
+
         if displayName != localUserState.displayName {
             displayName = localUserState.displayName
         }
-        self.cameraSwitchButtonPipViewModel.isDisabled = localUserState.cameraState.device == .switching
-        self.cameraSwitchButtonFullViewModel.isDisabled = localUserState.cameraState.device == .switching
+
+        self.cameraSwitchButtonPipViewModel.update(isDisabled: localUserState.cameraState.device == .switching)
+        self.cameraSwitchButtonPipViewModel.update(
+            accessibilityLabel: localUserState.cameraState.device.label(
+                fallbackLabel: self.cameraSwitchButtonPipViewModel.accessibilityValue ?? ""))
```

Comment:
This feels like a lot to process in one line of code. 
```suggestion
let fallback =  self.cameraSwitchButtonPipViewModel.accessibilityValue ?? ""
let label = localUserState.cameraState.device.label(fallbackLabel: fallback)
self.cameraSwitchButtonPipViewModel.update(accessibilityLabel: label)
```
I wonder if we really need the idea of the `fallbackLabel`?

Reply:
fallbackLabel is there because there isn't a label for each value in the enum.

<img width="519" alt="image" src="https://user-images.githubusercontent.com/73612854/154367841-9b52ad48-c777-41e3-8c51-82eb23a3f80e.png">

Reply:
I think i could change this to a nil/optional instead.

Reply:
Good catch, it's much simpler now.